packages/react-native-cli/src/lib/__test__/fixtures/AppDelegate-after-079.swift

```diff
@@ -0,0 +1,51 @@
+import Bugsnag
```

Comment:
just to confirm, this is the only change between these versions?

Reply:
There's also the `Bugsnag.start` call on line 18.

The reason we need new before/after fixture files is because the whole `AppDelegate.swift` template has changed in 0.79 with the introduction of `RCTReactNativeFactory`